cpp/oneapi/dal/algo/rbf_kernel/backend/gpu/compute_kernel_dense_dpc.cpp

```diff
@@ -15,60 +15,110 @@
 *******************************************************************************/
 
 #include "oneapi/dal/algo/rbf_kernel/backend/gpu/compute_kernel.hpp"
-#include "oneapi/dal/backend/interop/common_dpc.hpp"
-#include "oneapi/dal/backend/interop/table_conversion.hpp"
-
+#include "oneapi/dal/backend/primitives/reduction.hpp"
+#include "oneapi/dal/backend/primitives/blas.hpp"
+#include "oneapi/dal/backend/math.hpp"
 #include "oneapi/dal/table/row_accessor.hpp"
 
-#include <daal/src/algorithms/kernel_function/oneapi/kernel_function_rbf_kernel_oneapi.h>
-
 namespace oneapi::dal::rbf_kernel::backend {
 
 using dal::backend::context_gpu;
 using input_t = compute_input<task::compute>;
 using result_t = compute_result<task::compute>;
 using descriptor_t = detail::descriptor_base<task::compute>;
 
-namespace daal_rbf_kernel = daal::algorithms::kernel_function::rbf;
-namespace interop = dal::backend::interop;
+namespace pr = dal::backend::primitives;
 
 template <typename Float>
-using daal_rbf_kernel_t =
-    daal_rbf_kernel::internal::KernelImplRBFOneAPI<daal_rbf_kernel::defaultDense, Float>;
+sycl::event compute_rbf(sycl::queue& queue,
+                        const pr::ndview<Float, 1>& sqr_x,
+                        const pr::ndview<Float, 1>& sqr_y,
+                        pr::ndview<Float, 2> res_rbf,
+                        const Float coeff,
+                        const std::int64_t ld,
+                        const dal::backend::event_vector& deps) {
+    const Float* sqr_x_ptr = sqr_x.get_data();
+    const Float* sqr_y_ptr = sqr_y.get_data();
+    Float* res_rbf_ptr = res_rbf.get_mutable_data();
+
+    const Float threshold = dal::backend::exp_treshold<Float>();
+    const std::size_t l_dim = dal::detail::integral_cast<std::size_t>(ld);
```

Comment:
why `std::size_t` not `std::int64_t`?

Reply:
@PetrovKP, to use a single Integer type in the kernel